app/uk/gov/hmrc/economiccrimelevyreturns/models/email/AmendReturnSubmittedRequest.scala

```diff
@@ -22,7 +22,12 @@ final case class AmendReturnSubmittedParameters(
   name: String,
   dateSubmitted: String,
   periodStartDate: String,
-  periodEndDate: String
+  periodEndDate: String,
+  addressLine1: Option[String],
+  addressLine2: Option[String],
+  addressLine3: Option[String],
+  addressLine4: Option[String],
```

Comment:
The success screen converts the address object to a single block of html which is four or five lines long. Can't you ave a `Seq[String]` for all the lines and generate that sequence from the address object? You could have an extras parameter on the `Map` you pass to the emsilrenderer that says how many address lines there are.

Reply:
Yes that is correct, but this way it's bit more verbose and clearer what is happening, especially on renderer side. If you are ok with it I would like to keep it this way, even more if we take into the account that DC ticket is already merged in 😄